tests/test_pre_distort/CMakeLists.txt

```diff
@@ -11,9 +11,10 @@ set(APP_DEPENDENT_MODULES 	"lib_sw_dac")
 
 set(APP_PCA_ENABLE 			OFF)
 
-set(APP_COMPILER_FLAGS    	-O3
-							-g
-							-report)
+set(APP_COMPILER_FLAGS          -O3
+                                -mcmodel=large
```

Comment:
Formatting a bit off - suspect tabs vs spaces. Using spaces will help viewing on all editors (age old debate..)

Reply:
Yeah - I have been replacing all tabs with spaces, but this one must have escaped.